app-emacs/lv/files/lv-require-cl-lib.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lv.el b/lv.el
```

Comment:
Nit 1: I'd use [iwdevtools](https://github.com/ionenwks/iwdevtools)'s (app-portage/iwdevtools) scrub-patch to clean up the first two lines.

Nit 2: I'd name the file ```lv-${FIRST_VERSION_THIS_APPLIED_TO}-cl-lib.patch``` instead in case you have to rebase it in future.

Less-of-a-nit-and-matters-a-bit-more: Could you add a brief message at the top of the patch/file to explain why it's needed? See [here](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions) in particular (but the whole page is pretty useful/worth reading if haven't already).